libs/community/langchain_community/llms/bedrock.py

```diff
@@ -162,9 +163,19 @@ def prepare_output(cls, provider: str, response: Any) -> dict:
             else:
                 text = response_body.get("results")[0].get("outputText")
 
+        headers = response.get("ResponseMetadata", {}).get("HTTPHeaders", {})
+
         return {
             "text": text,
             "body": response_body,
+            "usage": {
+                "prompt_tokens": int(
+                    headers.get("x-amzn-bedrock-input-token-count", 0)
+                ),
+                "completion_tokens": int(
+                    headers.get("x-amzn-bedrock-output-token-count", 0)
+                ),
```

Comment:
This is great! I was looking to achieve the same thing to enable cost monitoring on the Anthropic Bedrock models. Should we also add total_tokens (sum of prompt_tokens + completion_tokens), to keep it compatible with the OpenAI model?

Reply:
Thanks for the comment! I added the `total_count`

Reply:
Thanks! You have linting errors, fyi.

Now need someone with write access to the project to look at it and give their review.